src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import FormLabel from '../FormLabel/FormLabel';
+import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
+import './RadioGroup.scss';
+
+/**
+ * Allows users to pick a value from predefined list of options.
+ * Ideally, it should be used when there are less than 5 options,
+ * otherwise you should consider using SelectInput instead.
+ */
+
+const propTypes = {
+  /**
+   * Radio group 'name'
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * Callback function to call on change event.
+   */
+  onChange: PropTypes.func.isRequired,
+  /**
+   * Options for radio group
+   */
+  options: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    isDisabled: PropTypes.bool,
+  })).isRequired,
+  /**
+   * Custom content to be displayed above the RadioGroup.
+   * Both a title and description can be included in the legend.
+   */
+  legend: PropTypes.shape({
+    title: PropTypes.node.isRequired,
```

Comment:
`title` shouldn't be required.

if we want to prevent someone from passing a `description` without a `title`, then we should make a "requires other prop" proptype and warn if the consumer sets a description without a title